src/evaluator/matchers/semver_gte.ts

```diff
@@ -0,0 +1,14 @@
+import { ILogger } from '../../logger/types';
+import { Semver } from '../../utils/Semver';
+
+export function greaterThanEqualToSemverMatcherContext(log: ILogger, ruleAttr: string) {
+  const ruleSemver = new Semver(ruleAttr);
+
+  return function greaterThanEqualToSemverMatcher(runtimeAttr: string): boolean {
+    const runtimeSemver = new Semver(runtimeAttr);
```

Comment:
What happen is Semver class throws an exception at this point?

Reply:
Matcher exceptions are handled in the `parser` function, which calls the matchers.

It's done this way to reduce code size, by handling and logging the errors in a single place.